src/parser/tera.pest

```diff
@@ -190,10 +190,10 @@ super_tag        = !{ variable_start ~ "super()" ~ variable_end }
 
 text       = ${ (!(block_start) ~ ANY)+ }
 
-raw_tag    = { tag_start ~ "raw" ~ tag_end }
-endraw_tag = { tag_start ~ "endraw" ~ tag_end }
-raw_text   = { (!endraw_tag ~ ANY)* }
-raw        = !{ raw_tag ~ raw_text ~ endraw_tag }
+raw_tag    = !{ tag_start ~ "raw" ~ tag_end }
+endraw_tag = !{ tag_start ~ "endraw" ~ tag_end }
+raw_text   = ${ (!endraw_tag ~ ANY)* }
+raw        = ${ raw_tag ~ raw_text ~ endraw_tag }
```

Comment:
I know i could go read the pest documentation, but could you maybe describe what this subtle change is doing?

Reply:
It just changes the way pest handle whitespace in those expression, before they were ignored and there we just enable it again.